src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuck.java

```diff
@@ -497,6 +504,22 @@ protected void cleanTmpDir(Path tmpDir) {
         }
     }
 
+    private void execute(CommandLine command, String stdin) {
+        InputStream inputStream = new ByteArrayInputStream(stdin.getBytes());
+
+        CollectingLogOutputStream outputStream = new CollectingLogOutputStream();
+
+        try {
+            Executor executor = new DefaultExecutor();
+            executor.setStreamHandler(new PumpStreamHandler(outputStream, outputStream, inputStream));
+            executor.execute(command);
+        } catch (IOException ee) {
```

Comment:
why ee rather than e? 

and should it be passed to NetCdfProcessingException when we rethrow NetCdfProcessingException